questionpy_server/repository/models.py

```diff
@@ -0,0 +1,65 @@
+from dataclasses import dataclass
+from datetime import datetime
+from functools import total_ordering
+from pathlib import Path
+
+from pydantic import BaseModel, PositiveInt
+from questionpy_server.utils.manfiest import ComparableManifest, SemVer
+
+
+class RepoMeta(BaseModel):
+    timestamp: datetime
+    size: PositiveInt
+    sha256: str
+
+
+@total_ordering
+class RepoPackageVersion(BaseModel):
+    version: SemVer
+    api_version: str
+
+    path: Path
```

Comment:
`Path` repräsentiert ja eigentlich einen lokalen Pfad. Wenn das trotzdem hilft, um die URL leichter zusammenzubauen, dann kann das ja verwendet werden. Allerdings sehe ich, dass du den `Path` an anderer Stelle zum `string` machst, um damit weiter arbeiten zu können. Von daher würde ich hier und in `RepoPackage` `string` verwenden.